Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/find_similar_incidents_test.py

```diff
@@ -181,6 +181,16 @@ def test_get_incidents_by_keys():
     assert res == incident_by_keys
 
 
+def test_build_similar_keys_list():
```

Comment:
Let's add a given-when-then here.
Other than that - approved

Reply:
dnoe